### DISCUSSION TITLE
技术资料与公开课分享 online only

### DISCUSSION THREAD

**Discussion by Lewisrxliu** [RoadSideUniversity/ClubHouse2024#39] [Announcements]
免责声明：分享人与以下分享内容的发布者组织者管理者没有任何利益关联，本着以自身学习的需要在学堂内部分享学习经验，自觉遵守所在地法律法规，自觉维护原内容创作者授权人及原使用者的正当权益，不转发，不推介，不交易，不做任何商业关联，参与方免除任何第三者关联责任，如需调整，请主动联系分享人自行选择规避或删除或更改相关内容。

https://docs.rapids.ai/install

BUILD / Streamlit + Snowflake community （Dec 5-6）
https://www.snowflake.com/build/americas/

https://goo.gle/CloudTechnicalSeries-Nov23

Future3 Campus-万向区块链实验室和HashKey Capital共同发起的Web3.0创新孵化平台http://www.future3.io

https://blockchainfestival.asia/ Blockchain Festival Singapore 2024 
⠀

**Comment by Lewisrxliu** [comment]
阿里云模型服务 灵积
http://dashscope.aliyun.com

**Reply by Lewisrxliu** [reply]
#阿里云新加坡创新加速器/创新中心
https://sg.alibabacloud.com/innovation-accelerator/index
#阿里云创新方案及客户成功案例
https://www.alibabacloud.com/customers
#阿里云China Gateway2.0
https://sg.alibabacloud.com/solutions/china-gateway
#阿里云学院
https://edu.alibabacloud.com/
#阿里云橙盟出海服务联盟
https://www.alibabacloud.com/zh/go_global/go-global-alliance-with-alibaba-cloud

**Comment by guttasingapore** [comment]
2023 参考书单
https://github.com/RoadSideUniversity/101/blob/main/2023%20booklist

**Comment by guttasingapore** [comment]
[Uploading 对中美冲突中经济制裁的分析报告（2023.12）.pdf…]()
